Sources/TestingMacros/Support/Additions/DeclGroupSyntaxAdditions.swift

```diff
@@ -8,7 +8,11 @@
 // See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 //
 
+#if swift(>=5.11)
+import SwiftSyntax
+#else
 public import SwiftSyntax
+#endif
```

Comment:
Are you sure we actually need these `#if swift(>=5.11)` guards? My understanding was 5.10 was not really affected, only an earlier range of 5.11-series toolchains were

Reply:
Yes, I checked: if you use the latest 5.10 toolchain, you will get an error here without the `public` keyword.